### PR TITLE
cleanup code

### PR DESCRIPTION
**Pull request by Kvicii**
Motivation:

cleanup code.

Modification:

1.fix wildcard import
2.for branch add break.

### CODE REVIEW

**Review by hyperxpro — APPROVED** [review]

File: codec/src/main/java/io/netty/handler/codec/compression/Bzip2BlockCompressor.java [hunk 1]
```diff
@@ -113,6 +115,7 @@ private void writeSymbolMap(ByteBuf out) {
             for (int j = 0, k = i << 4; j < HUFFMAN_SYMBOL_RANGE_SIZE; j++, k++) {
                 if (blockValuesPresent[k]) {
                     condensedInUse[i] = true;
+                    break;
```
Comment (normanmaurer) [inline codec/src/main/java/io/netty/handler/codec/compression/Bzip2BlockCompressor.java:118]: Are you sure this is correct ? /cc @idelpivnitskiy

**Review by normanmaurer — CHANGES_REQUESTED** [review]

File: codec/src/main/java/io/netty/handler/codec/compression/Bzip2BlockCompressor.java (hunk 1 above)
Comment (chrisvest) [reply codec/src/main/java/io/netty/handler/codec/compression/Bzip2BlockCompressor.java:118]: I agree, this looks very suspicious.

Comment (idelpivnitskiy) [reply codec/src/main/java/io/netty/handler/codec/compression/Bzip2BlockCompressor.java:118]: Ufff, it's been years... I would be hesitant to make this change without justifying test that can demonstrate incorrect behavior without `break`.

Comment (normanmaurer) [inline codec/src/main/java/io/netty/handler/codec/compression/Bzip2BlockCompressor.java:118]:
```suggestion
```

**Review by normanmaurer — APPROVED** [review]

**Comment by normanmaurer** [comment]
I reverted the `break` change...

**Comment by normanmaurer** [comment]
@Kvicii thanks!

**Pull request merged**